README.md

```diff
@@ -14,7 +14,7 @@
 
 
 <p align="center">
-  <a href="https://www.algolia.com/doc/api-client/getting-started/install/go/" target="_blank">Documentation</a>  •
+  <a href="https://www.algolia.com/doc/framework-integration/django/options/?language=python" target="_blank">Documentation</a>  •
```

Comment:
Hi @sxalexander thank you for your contribution.
I've updated the link to rather point to the first page of the documentation (the setup page)

```suggestion
  <a href="https://www.algolia.com/doc/framework-integration/django/setup/?client=python" target="_blank">Documentation</a>  •
```